### PR TITLE
Substitute PR request for PR requested reviewers request in `$assignRandomReviewer` action builtin

### LINKED ISSUE #34: Remove unnecessary pull request request
**Is your feature request related to a problem? Please describe.**
Currently, in the `$assignRandomReviewer` action builtin, we perform a pull request request just to get the requested reviewers when GitHub API provides a specific request for this.

**Describe the solution you'd like**
Instead of using a generic request (pull request request) just to get the pull request requested reviews, use https://docs.github.com/pt/rest/pulls/review-requests#list-requested-reviewers-for-a-pull-request

**Describe alternatives you've considered**
*None*

**Additional context**
*None*

### PR DESCRIPTION
**Pull request by shay2025**
This pull request introduces the replacement of the get pull request request in `$assignRandomReviewer` action builtin that is used to get the requested reviewers when we can specify the request to get the requested reviewers directly.

Closes #34.

### CODE REVIEW

**Comment by shay2025** [comment]
@marcelosousa after a user makes an approval to a pull request they won't be taken into account in the ListReviewers response. So right now that means that with `$assignRandomReviewer` exists the possibility that it can be re-requested a review.

File: utils/github.go [hunk 1]
```diff
@@ -138,3 +138,29 @@ func GetPullRequestFiles(ctx context.Context, client *github.Client, owner strin
...
 }
+
+func GetPullRequestReviewers(ctx context.Context, client *github.Client, owner string, repo string, number int, opts *github.ListOptions) (*github.Reviewers, error) {
+	fs, err := PaginatedRequest(
+		func() interface{} {
+			return &github.Reviewers{}
+		},
+		func(i interface{}, page int) (interface{}, *github.Response, error) {
+			fls := i.(*github.Reviewers)
+			fs, resp, err := client.PullRequests.ListReviewers(ctx, owner, repo, number, &github.ListOptions{
```
Comment (ferreiratiago) [inline utils/github.go:149]: Please rename `fs` to `reviewers`.

File: utils/github.go [hunk 2]
```diff
@@ -138,3 +138,29 @@ func GetPullRequestFiles(ctx context.Context, client *github.Client, owner strin
 
 	return fs.([]*github.CommitFile), nil
 }
+
+func GetPullRequestReviewers(ctx context.Context, client *github.Client, owner string, repo string, number int, opts *github.ListOptions) (*github.Reviewers, error) {
+	fs, err := PaginatedRequest(
```
Comment (ferreiratiago) [inline utils/github.go:143]: Please rename `fs` to `reviewers`.

**Review by ferreiratiago — CHANGES_REQUESTED** [review]
Some rename required. Please have a look.

[reviewpad-sig]: <>
[View on **Reviewpad**](https://beta.reviewpad.com/review/github.com/reviewpad/reviewpad/pull/35?codereviewid=ecdf30dd-14f3-4b57-94b0-1031a40efae9)

**Review by ferreiratiago — APPROVED** [review]
LGTM

**Pull request merged**